c2corg_ui/static/partials/suggestionusers.html

```diff
@@ -0,0 +1,6 @@
+  <span class="suggestion-text"> 
```

Comment:
In a recent PR I have introducted a "_" in the "suggestion" partials names.
To be consistent this partial should be renamed to ``suggestion_users.html``.
See https://github.com/c2corg/v6_ui/tree/master/c2corg_ui/static/partials